SYCL/DeviceLib/cmath_fp64_test.cpp

```diff
@@ -97,8 +95,8 @@ template <class T> void device_cmath_test(s::queue &deviceQueue) {
 
         res_access[i++] = !(std::signbit(infinity) == 0);
         res_access[i++] = !(std::signbit(minus_infinity) != 0);
-        res_access[i++] = !(std::signbit(nan) == 0);
-        res_access[i++] = !(std::signbit(minus_nan) != 0);
+        // res_access[i++] = !(std::signbit(nan) == 0);
+        // res_access[i++] = !(std::signbit(minus_nan) != 0);
```

Comment:
If the code is not needed it is better to delete it

Reply:
Done.